Sources/TuistGenerator/Generator/GeneratedProject.swift

```diff
@@ -37,9 +37,7 @@ final class GeneratedProject {
     /// - Parameter path: Path to the project (.xcodeproj)
     /// - Returns: GeneratedProject instance.
     func at(path: AbsolutePath) throws -> GeneratedProject {
-        let xcode = try XcodeProj(pathString: path.pathString)
```

Comment:
Initially before testing on a RAM Disk, I was testing on Google Drive :p - and there I found this step fails sometimes as the generated file fails to be read from disk (if done immediately).